### ISSUE TITLE
First draft of the overview page in the wiki

### ISSUE THREAD

**Issue by superkhau** [strongloop/loopback-next#103]
See https://github.com/strongloop/loopback-next/wiki/Overview

- [x] Clean up the wiki page
- [x] Fill in Overview table with terms and simple descriptions
- [x] Update wiki page contents
  - [x] Fix up architecture image @ritch @superkhau
    - Replaced image with simple array diagram for now
  - [x] Create issues to fill in content for individual construct sections
    - #197 #198 #199 #200 #201 #202 #203 #204 #205 #206
- [x] Create an [issue](https://github.com/strongloop/loopback-next/issues/185) for @crandmck to migrate contents to LoopBack.io @superkhau

**Comment by superkhau** [comment]
Blocked until I can set up meeting with @ritch for first run (and then confirm with @raymondfeng @bajtos during next LBn sync up)

**Comment by superkhau** [comment]
Still blocked, managed to get single sentence summary descriptions though. Will try to get some agreement on terminology and a first draft of the diagram from @raymond/@ritch.

**Comment by jonathan-casarrubias** [comment]
@superkhau I just want to be entirely sure about the architecture diagram, so I need to ask a question.

Since I don't see the `component` concept anymore, I'm assuming that within the dynamic bindings section there is an `extension` artifact, Is that the way we are going to extend the framework?

If so, I would like to see an example of it as soon as possible, I still haven't see a unit test for that but  it would be great to start porting the modules I maintain.

Regards!
Jon

**Comment by superkhau** [comment]
@jonathan-casarrubias The picture is out of date. We are working on the extensibility story this sprint actually, so you'll see something probably by the end of the week.

TLDR, you're going to be able to "hook" into the framework via special keywords + binding into the context. It's too early to provide you anything now, but keep an eye on some releated issues:

- https://github.com/strongloop/loopback-next/issues/155
- https://github.com/strongloop/loopback-next/issues/156

**Comment by jonathan-casarrubias** [comment]
@superkhau thanks for your response, I will subscribe to these related issues: I'm expecting to be one of the very first devs to extend the framework so I'll be looking forward these functionalities.

Cheers!
Jon

**Comment by superkhau** [comment]
@jonathan-casarrubias Wonderful, I'll ping you as soon as we get something basic in. You can help us dogfood the API to make sure it fits the various use cases of your current component set.

**Comment by superkhau** [comment]
Paired with @ritch to fill our the rest of https://github.com/strongloop/loopback-next/wiki/Overview -- it's good enough for now. Closing.
